core/vm/actor/builtin/v0/storage_power/storage_power_actor_state.hpp

```diff
@@ -39,34 +56,52 @@ namespace fc::vm::actor::builtin::v0::storage_power {
 
   struct State {
     static State empty(IpldPtr ipld);
+
     outcome::result<void> addToClaim(const Address &miner,
                                      const StoragePower &raw,
                                      const StoragePower &qa);
     outcome::result<void> addPledgeTotal(const TokenAmount &amount);
     outcome::result<void> appendCronEvent(const ChainEpoch &epoch,
                                           const CronEvent &event);
+    void updateSmoothedEstimate(int64_t delta);
+    std::tuple<StoragePower, StoragePower> getCurrentTotalPower() const;
```

Comment:
```suggestion
    auto getCurrentTotalPower() const;
```

Reply:
You could use it, but not mandatory. Just FYI that you could make it more easy

Reply:
I prefer to use explicit result declaration in header files, so I can find return type without opening .cpp.